apps/voting/app/src/components/VoteActions.js

```diff
@@ -0,0 +1,271 @@
+import React, { useState, useCallback } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  ExternalLink,
+  IconCheck,
+  IconCross,
+  Info,
+  GU,
+  RADIUS,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import { useAppState, useConnectedAccount } from '@aragon/api-react'
+import { noop, formatDate } from '../utils'
+import { useExtendedVoteData } from '../vote-hooks'
+import { VOTE_NAY, VOTE_YEA } from '../vote-types'
+import { isVoteAction } from '../vote-utils'
+
+const VoteActions = React.memo(({ vote, onVoteYes, onVoteNo, onExecute }) => {
+  const theme = useTheme()
+  const connectedAccount = useConnectedAccount()
+  const { tokenSymbol } = useAppState()
+  const [changeVote, setChangeVote] = useState(false)
+  const handleChangeVote = useCallback(() => setChangeVote(true), [])
+
+  const { connectedAccountVote, data } = vote
+  const { snapshotBlock, startDate, open } = data
+  const { canUserVote, canExecute, userBalance } = useExtendedVoteData(vote)
+  const hasVoted = [VOTE_YEA, VOTE_NAY].includes(connectedAccountVote)
+
+  if (!open) {
+    return (
+      <React.Fragment>
+        {canExecute && isVoteAction(vote) && (
+          <React.Fragment>
+            <Button mode="strong" onClick={onExecute} wide>
+              Enact this vote
+            </Button>
+            <Info>
+              The voting period is closed and the vote has passed.{' '}
+              <span css="font-weight: bold;">Anyone</span> can now enact this
+              vote to execute its action.
+            </Info>
+          </React.Fragment>
+        )}
+        {hasVoted && (
+          <div
```

Comment:
We probably want to center this with the content cc @dizzypaty?

This is how it currently looks:

<img width="817" alt="Screen Shot 2019-08-09 at 2 35 34 PM" src="https://user-images.githubusercontent.com/4166642/62779255-f7738980-bab2-11e9-9ceb-f2c209341830.png">

Reply:
Yes, the content should be centered. I also changed the copy to make the action clearer, making this module a bit more compact and reduced the icon size. 

Should look like this: 

<img width="1534" alt="VOTING - PASSED" src="https://user-images.githubusercontent.com/7205369/62856812-32193400-bcf6-11e9-8798-a4c2ba2dfe39.png">

Reply:
@sohkai I thought the date was not easy to fetch (pinged you in Figma about this)

Reply:
Yes, we should remove the date from this.